picoCTF-shell/hacksport/deploy.py

```diff
@@ -642,9 +642,11 @@ def url_for(web_accessible_files,
     logger.debug("...Created service files '%s','%s'.", service_file,
                  socket_file)
 
+
     # template the description
+    # change newline for <br>, otherwise it won't render on the pico website
     problem.description = template_string(problem.description,
-                                          **get_attributes(problem))
+                                          **get_attributes(problem)).replace('\n','<br>')
```

Comment:
@hiliang-cmu Would this solution work with the game? (It seems `<br>` would then be rendered as plaintext...)

Reply:
Well one side or the other would have to do a replacement between `\n` and `<br>`

Since the game already has some string replacement going on, would probably be more natural to put it there

Reply:
@carlislemc to clarify, I meant the two client sides: a plain HTML interface and the game interface.

This would render as-is then in the former, and the latter already has other string replacements going on that we can add to.

So this is fine serverside, and we'll make note to add a `<br`> to `\n` replacement in the game.